RUNBOOK.md

```
@@ -0,0 +1,297 @@
+docker tag bugspotter-api:latest "bugspotter-api:pre-$short" 2>/dev/null || true
+docker tag "$IMAGE" bugspotter-api:latest
+cd /opt/bugspotter
+docker compose up -d --no-build api worker
+
+printf '%s' "$new_digest" > "$STATE_FILE"
+echo "$(date -Iseconds) deployed $new_digest" >> "$LOG_FILE"
+```
+
+Crontab (installed for the `deploy` user, not root):
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)